docs/references/what_is_a_usn.md

```diff
@@ -0,0 +1,8 @@
+# What is a USN ?
+
+A USN (Ubuntu Security Notice) is the way for Canonical to public catalog and display security
```

Comment:
```suggestion
An Ubuntu Security Notice (USN) is the way that Canonical publicly catalogues and displays security
```

Reply:
I think we ought to also include this link https://ubuntu.com/security/notices

Reply:
Good point